src/main/java/org/apache/sling/api/resource/observation/package-info.java

```diff
@@ -17,7 +17,7 @@
  * under the License.
  */
 
-@Version("1.2.2")
+@Version("1.3.0")
```

Comment:
I assume `bnd` requested this change. But it it really needed? I think adding new constants to an interface would not break implementors of that interface, so this should be 1.2.3 instead.

Reply:
Agreed, I added a similar comment to the Jira issue

Reply:
compare with https://github.com/bndtools/bnd/issues/4197. It might break compatibility in certain edge cases!